src/Pages/MyCollection/MyCollection.tsx

```diff
@@ -3,19 +3,19 @@ import Phono_Logo from '../../assets/Phono_Logo';
 import SearchBar from '../../components/SearchBar/SearchBar';
 import AlbumInfo from '../../components/AlbumInfo/AlbumInfo';
 import NavBar from '../../components/NavBar/NavBar';
-import { useState } from 'react';
-import useMyCollection from '../../utils/useMyCollection';
+import { useEffect, useState } from 'react';
+import useCollection from '../../utils/useCollection';
 import CoverSwiper from '../../components/CoverSwiper/CoverSwiper';
-import NoMatchingSearchResult from './MyCollectionAssets/NoMatchingSearchResult.svg';
+import NoMatchingSearchResult from '../../assets/NoMatchingSearchResult.svg';
```

Comment:
I got the recommendation to make a component out of the svg and import it instead. Maybe you want to do the same?